### PR TITLE
Feat: Added useContext in ecom app

### PR DESCRIPTION
**Pull request by sakleprajwal**
useContext is added in spacTrend ecom app.

https://user-images.githubusercontent.com/40752468/160276389-07d1397b-699e-4342-97eb-7620417268a5.mp4

### CODE REVIEW

**Review by kotalwar-krishna — APPROVED** [review]
Looks fine

File: src/components/Categories/Categories.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
+import axios from 'axios';
 import React from 'react';
-import "../../styles/App.css";
 import { Link } from 'react-router-dom';
-import { useState, useEffect } from 'react';
-import axios from 'axios';
+import { useState } from 'react';
+import "../../styles/App.css"
+import { useEffect } from 'react';
+import { useContext } from 'react';
```
Comment (rahulcs754) [inline src/components/Categories/Categories.js:7]: you can access in one line 
import {useState , useEffect} from "react";

File: src/hooks/cart-context/cart-context.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const cartContext = createContext({itemsInCart: 4});
+
+const CartProvider = ({children}) => {
+    const [itemsInCart, setItems] = useState(0)
+    const addToCart = () => {
+        setItems(itemsInCart+1)
+    }
+    const removeFromCart = () => {
+        setItems(itemsInCart-1)
+    }
```
Comment (rahulcs754) [inline src/hooks/cart-context/cart-context.js:14]: i think you should have store id in state no number

File: src/hooks/wishlist-context/wishlist-context.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+const wishlistContext = createContext({itemsInWishlist: 4});
+
+const WishlistProvider = ({children}) => {
+    const [itemsInWishlist, setItems] = useState(0)
+    const addToWishlist = () => {
+        setItems(itemsInWishlist+1)
+    }
+    const removeFromWishlist = () => {
+        setItems(itemsInWishlist-1)
+    }
```
Comment (rahulcs754) [inline src/hooks/wishlist-context/wishlist-context.js:14]: you should store product id in wishlist

**Review by rahulcs754 — COMMENTED** [review]
after checking your code,
you have to add product id into cart and wishlist that will helpfull for remove product from cart and wishlist

**Review by Mahi24-prog — APPROVED** [review]
Looks Good go ahead

File: src/hooks/wishlist-context/wishlist-context.js (hunk 3 above)
Comment (sakleprajwal) [reply src/hooks/wishlist-context/wishlist-context.js:14]: There is no use of product ids **right now**. That's why I haven't stored it.
I will add this functionality in a separate pull request. Thanks for the review.

**Pull request merged**